aqueduct/flow.py

```diff
@@ -302,6 +315,7 @@ def _run_tasks(self):
         self._tasks.append(asyncio.ensure_future(self._check_is_alive()))
 
         self._metrics_manager.start(queues_info=self._get_queues_info())
+        self._metrics_manager.collector.add_flow_steps_memory_usage(self._get_flow_steps_memory_usage())
```

Comment:
It seems run only once at start. For regular run you need add `while True` inside `_get_flow_steps_memory_usage` with some sleep time

Reply:
Is it correct to replace `while True` with `while self.state != FlowState.STOPPED`?